### PR TITLE
gnrc_ipv6_nib: doc: use attention tag instead of bold note

### PR DESCRIPTION
**Pull request by miri64**
As pointed out in https://github.com/RIOT-OS/RIOT/pull/7212#discussion_r123271139

### CODE REVIEW

File: sys/include/net/gnrc/ipv6/nib/conf.h [hunk 1]
```diff
@@ -167,8 +167,8 @@ extern "C" {
 /**
  * @brief   Number of default routers in the default router list.
  *
- * @note    **This number has direct influence on the maximum number of
- *          default routers**
+ * @attention   This number has direct influence on the maximum number of
+ *              default routers**
```
Comment (smlng) [inline sys/include/net/gnrc/ipv6/nib/conf.h]: remove trailing `**`, too

File: dist/tools/doccheck/check.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 RIOTBASE=$(readlink -f "$(dirname $(realpath $0))/../../..")
 
-ERRORS=$(make doc 2>&1 | grep '.*warning' | sed "s#${PWD}/\([^:]*\)#\1#g")
+ERRORS=$(make -C ${RIOTBASE} doc 2>&1 | grep '.*warning' | sed "s#${PWD}/\([^:]*\)#\1#g")
```
Comment (smlng) [inline dist/tools/doccheck/check.sh]: why this change, seems unrelated?

Comment (miri64) [inline dist/tools/doccheck/check.sh]: True. Is from my tests for #7217. Will remove.
